README.md

```diff
@@ -349,6 +349,19 @@ partition = PartitioningScheme.assign(partitions, event)
 producer.produce(event, topic: "events", partition: partition)
 ```
 
+Another option is to configure a custom client partitioner that implements `partition_for_key(partition_count, message)` and uses the same schema as the other client. For example:
+
+```ruby
+class CustomPartitioner
+  def partition_for_key(partition_count, message)
+    ...
+  end
+end
+  
+partitioner = CustomPartitioner.new
+Kafka.new(partitioner: partitioner, ...)
```

Comment:
Looking at this, I think we should use `call` instead of `partition_for_key`. That would allow passing Procs instead of having to add a new class – a more Rubyesque approach in my view.

```ruby
partitioner = -> (partition_count, message) -> { ... }
Kafka.new(partitioner: partitioner, ...)
```